pkg/services/cosmosdb/sql-api-provision.go

```diff
@@ -33,11 +33,9 @@ func (s *sqlAccountManager) deployARMTemplate(
 	}
 
 	p := s.buildGoTemplateParams(pp, dt, "GlobalDocumentDB")
-	p["capability"] = "EnableTable"
 	if instance.Tags == nil {
```

Comment:
Instead of just removing this, is there some other value that should be set instead? tbh, I don't understand the who "capability" aspect of Cosmos like you do.